### PR TITLE
release v8.1.1

### PR DESCRIPTION
**Pull request by xh3b4sd**
We want to get some PM fixes out so people can move on with Node Pools. I figured there is not too much relevant to be mentioned in the changelogs. I omitted some minor refactoring PRs that do not provide a lot of value for the user. Please have a look and let me know if this is ok. Goal here is to tag the operator, get the release notes sorted and have something final to test prior to making the release official.

### CODE REVIEW

File: pkg/project/project.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "8.1.1-dev"
+	bundleVersion        = "8.1.1"
```
Comment (kopiczko) [inline pkg/project/project.go]: This is a release PR which IMO should be stated in the PR title. Otherwise it's very easy to miss that.

Comment (xh3b4sd) [reply pkg/project/project.go]: Can you check if the title works for you now. If not please suggest something.

Comment (kopiczko) [reply pkg/project/project.go]: IMO this should be simply `release v8.1.1`. You can write in the description why you create the release.

Comment (xh3b4sd) [reply pkg/project/project.go]: Sorted. You happy now?

**Comment by xh3b4sd** [comment]
@giantswarm/team-ludacris cool with this?

File: pkg/project/version_bundle.go [hunk 2]
```diff
@@ -9,84 +9,18 @@ func NewVersionBundle() versionbundle.Bundle {
...
-				Kind:        versionbundle.KindFixed,
-				URLs: []string{
-					"https://github.com/giantswarm/aws-operator/pull/2110",
-				},
-			},
-			{
-				Component:   "cloudformation",
-				Description: "Drain nodes when deleting Node Pools.",
+				Component:   "aws-operator",
+				Description: "Fix network traffic between Node Pools.",
```
Comment (tuommaki) [inline pkg/project/version_bundle.go]:
```suggestion
				Description: "Allow network traffic between Node Pools.",
```
Minor thing, but possibly important. External person might not know what was wrong and therefore what was fixed.

File: pkg/project/project.go (hunk 1 above)
Comment (kopiczko) [reply pkg/project/project.go]: Yes.

**Pull request merged**